recipes/jaxsim/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "jaxsim" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ami-iit/jaxsim/archive/v{{ version }}.tar.gz
```

Comment:
Please build from the sdist from PyPI. Then you wouldn't need the `SETUPTOOLS_SCM_PRETEND_VERSION` below.

Reply:
Thanks @xhochy, solved in 12f17c3a7fe9e2ce2d1895de3e7eef84b659c1b2